docs/guides/admin/docs/configuration/workflow.md

```diff
@@ -84,6 +88,10 @@ Start by naming the workflow and giving it a meaningful description:
   installations). If there are two workflows with the same id, the one corresponding to the user’s organization is
   always chosen. This pertains workflow dropdowns (for example, the “Add new event” dropdown) as well as workflows
   included in other workflows via the `include` workflow operation handler.
+* The `roles` define which user roles are allowed to see and start this workflow. If this is omitted or no roles are
+  specified, everyone can see and start the workflow (provided the `organization` constraints are satisfied). Also,
+  users with `ROLE_ADMIN` can see and start every workflow. Note that the workflows included in Opencast do not set
+  roles.
```

Comment:
Maybe add that any user that has _one_ of the specified roles can start the workflow instead of needing to have _all_ the listed roles?

Reply:
Good point, fixed!